### ISSUE TITLE
equals should have @Override annotation

### ISSUE THREAD

**Issue by HelgeStenstrom** [goxr3plus/java-google-speech-api#7]
https://github.com/goxr3plus/java-google-speech-api/blob/1863c920b637831f71d1d49f6f04a9396443fede/src/main/java/com/goxr3plus/speech/util/Complex.java#L116

because it does override the method in Object.

**Comment by goxr3plus** [comment]
Yep, i haven't seen it, cause i forked this project and modified :)

**Comment by HelgeStenstrom** [comment]
Well, it doesn't affect the behavior of the running code. But it's good to annotate @Override for the sake of the reader of the code. And if you @Override something that isn't in the interface or parent class, you'll get an error.

**Comment by goxr3plus** [comment]
I have added you as a collaborator on this repository => https://github.com/goxr3plus/java-google-speech-api/invitations

You have direct access to master , please be careful on your commits :)
